code/controllers/subsystem/mobs.dm

```diff
@@ -9,6 +9,7 @@ SUBSYSTEM_DEF(mobs)
 	var/static/list/clients_by_zlevel[][]
 	var/static/list/dead_players_by_zlevel[][] = list(list()) // Needs to support zlevel 1 here, MaxZChanged only happens when CC is created and new_players can login before that.
 	var/static/list/cubemonkeys = list()
+	var/giant_spiders = 0
```

Comment:
```suggestion
	/// The amount of giant spiders that exist in the world. Used for mob capping.
	var/giant_spiders = 0
```

Also finally a var scoped in the right place. This isn't aimed at you, just my hatred for people making globals when something can be scoped better.